src/asm/parser.y

```diff
@@ -920,64 +934,75 @@ dl:
 
 def_equ:
 	def_id POP_EQU const {
-		sym_AddEqu($1, $3);
+		$$ = std::move($1);
+		sym_AddEqu($$, $3);
```

Comment:
Why not

```suggestion
		sym_AddEqu($1, $3);
		$$ = std::move($1);
```

?

Reply:
Is there any reason *to* do that though? The parser rules are consistently written to do `$$ = std::move(...)` ASAP to avoid making this decision every time.

Reply:
Ah, fair.